cmd/cli/plugin/package/package_install.go

```diff
@@ -25,8 +25,9 @@ var packageInstallCmd = &cobra.Command{
 	
     # Install package contour with kubeconfig flag and waiting for package reconciliation to complete	
     tanzu package install contour-pkg --package-name contour.tanzu.vmware.com --namespace test-ns --version 1.15.1-tkg.1-vmware1 --kubeconfig path/to/kubeconfig`,
-	RunE:         packageInstall,
-	SilenceUsage: true,
+	RunE:              packageInstall,
+	SilenceUsage:      true,
+	PersistentPreRunE: packageMetadataAvailabilityCheck,
```

Comment:
I think ```"package install"``` and ```"package installed"``` operation should use the same availability call (they need both ```"packagemetadatas"``` and ```"packageinstalls"``` api to be available). But right now, ```"package install"``` is using ```"packageMetadataAvailabilityCheck"``` and ```"package installed"``` operations are using ```"packageInstallAvailabilityCheck"```.

Reply:
thanks for the review @maralavi , now we put all the checks together